src/powershell/install-category.ps1

```diff
@@ -0,0 +1,23 @@
+$category = $env:CATEGORY
```

Comment:
We should have all of our environment variables follow the same case format:
```suggestion
$category = $env:Category
```

Note that you'll need to make changes elsewhere to match this change.

Reply:
@dav3r resolved with [23b96f5](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/23b96f567572aa03299c91c979afc6a6506cb0cc)